src/MentionWrapper.js

```diff
@@ -183,19 +183,29 @@ class MentionWrapper extends Component {
     } = this.props;
     const { active, child, left, top, options } = this.state;
     const { item, className, style } = child;
-    
+
+    const componentProps = {
+      ...inputProps,
+      ref: this.inputRef,
+      onBlur: this.handleBlur,
+      onInput: this.handleInput,
+      onKeyDown: this.handleKeyDown
+    };
+
+    let inputElement;
+    if (component) {
+      const Component = component;
```

Comment:
input props should have a CustomInputComponent, lets not use the pre-existing 'component' field...